massa-network/src/handshake_worker.rs

```diff
@@ -39,35 +41,57 @@ pub struct HandshakeWorker {
 impl HandshakeWorker {
     /// Creates a new handshake worker.
     ///
+    /// Manage a new connection and perform a normal handshake
+    ///
+    /// Used for incomming and outgoing connections.
+    /// Add the node with
```

Comment:
It looks like this sentences does not have a end ?

Reply:
Oups

Reply:
Fixed